service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/PlacementController.java

```diff
@@ -111,4 +124,13 @@ private void removeTemporaryFields(CaseDetails caseDetails, String... fields) {
             caseDetails.getData().remove(field);
         }
     }
+
+    private void publishPlacementApplicationUploadEvent(CallbackRequest callbackRequest, RequestData requestData) {
+        applicationEventPublisher.publishEvent(
+            new PlacementApplicationEvent(callbackRequest, requestData.authorisation(), requestData.userId()));
+    }
+
+    private Boolean isUpdatingExistingPlacement(Placement previousPlacement, Placement newPlacement) {
```

Comment:
any reason why we are preferring `Boolean` and not `boolean`?

Reply:
Good spot, typo will change to 'boolean'